pcdsdevices/mirror.py

```diff
@@ -884,6 +884,58 @@ def transmission(self):
         return 1
 
 
+@reorder_components(
+    end_with=['coating', 'x', 'y', 'pitch', 'bender_us', 'bender_ds',
+        'x_enc_rms', 'y_enc_rms', 'pitch_enc_rms', 'bender_us_enc_rms',
+        'bender_ds_enc_rms', 'us_rtd', 'ds_rtd']
```

Comment:
The CI is upset about the formatting of all of these lists and all of the "doc=" blocks because the indent doesn't match. We've been formatting like this when we have a huge list of things to include:
```suggestion
    end_with=[
        'coating', 'x', 'y', 'pitch', 'bender_us', 'bender_ds',
        'x_enc_rms', 'y_enc_rms', 'pitch_enc_rms', 'bender_us_enc_rms',
        'bender_ds_enc_rms', 'us_rtd', 'ds_rtd',
    ]
```

Reply:
Fixed Style